src/Microsoft.DotNet.Wpf/src/PresentationFramework/Microsoft/Win32/CommonItemDialog.cs

```diff
@@ -353,7 +353,7 @@ internal bool MessageBoxWithFocusRestore(string message,
             bool ret = false;
 
             // Get the window that currently has focus and temporarily cache a handle to it
-            IntPtr focusHandle = UnsafeNativeMethods.GetFocus();
+            IntPtr focusHandle = MS.Win32.UnsafeNativeMethods.GetFocus();
```

Comment:
Global usings as called out above to resolve this without changing this file.

Reply:
Unfortunately global usings don't always fix these, so this one may need to stay